src/lists/Spring2019List.js

```diff
@@ -36,13 +36,12 @@ export default [
     },
     {
         id: 3,
-        image: ChantalleImage,
-        name: "Chantalle Bril",
-        title: "Secretary",
-        description: "Takes meeting minutes and sends out emails. Responsible for all forms of communication.",
-        email: "cbril@csu.fullerton.edu",
-        linkedin: "https://www.linkedin.com/in/chantallebril/",
-        github: "https://github.com/cbril"
+        image: JaneenImage,
+        name: "Janeen Yamak",
+        title: "Comm. Coordinator",
```

Comment:
Any reason why Communications was shortened? Everything else is expanded

Reply:
I was under the assumption that it was so long that it was stretching the student cards out, but looking now that doesn't seem to be the case. Fixed